dags/airflow_monitoring.py

```diff
@@ -15,7 +15,7 @@
     default_args=default_args,
     description='liveness monitoring dag',
     schedule_interval=None,
-    dagrun_timeout=timedelta(minutes=60))
+    dagrun_timeout=timedelta(minutes=20))
```

Comment:
Should we even be managing this DAG at all? As it's coming from the composer environment itself.

> The liveness DAG is stored in the dags/ folder and visible in the Airflow UI. The frequency and contents of the liveness DAG are immutable and must not be modified. Changes to the liveness DAG do not persist.

https://cloud.google.com/composer/docs/composer-2/monitor-environments

Reply:
I agree with this. I would prefer to delete this dag from this repo... we just have to keep it in the exclusion list in `gsutil rsync` calls, e.g.

```
gsutil -m rsync -x 'airflow_monitoring|.*\.pyc$' -cdr "$LOCAL_DAGS_FOLDER" "$DAG_GCS_PREFIX"
```